### PR TITLE
[PT-BR] include.md, jsxFragmentFactory.md and strictNullChecks.md

### PR DESCRIPTION
**Pull request by hmartiins**
I translated the following files from the tsconfig session: 

- include.md; 
- jsxFragmentFactory.md;
- strictNullChecks.md.

@orta @khaosdoctor @alvarocamillont @danilofuchs

### CODE REVIEW

File: packages/tsconfig-reference/copy/pt/options/jsxFragmentFactory.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+display: "jsxFragmentFactory"
+oneline: "Faz alguma coisa"
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/jsxFragmentFactory.md:3]: @orta is this a TODO?

File: packages/tsconfig-reference/copy/pt/options/jsxFragmentFactory.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+---
+
+Especifique a função de fábrica de fragmentos JSX para ser acionada ao direcionar o react JSX emitir com a opção do compilador `jsxFactory` especificada, por exemplo `Fragment`.
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/jsxFragmentFactory.md]:
```suggestion
Especifique a função que será a fábrica de fragmentos JSX para ser acionada ao usar o react JSX com a opção do compilador `jsxFactory` especificada, por exemplo `Fragment`.
```

File: packages/tsconfig-reference/copy/pt/options/jsxFragmentFactory.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+Esta opção pode ser usada como base por arquivo um muito semelhante a [Babel's `/** @jsxFrag h */` directive](https://babeljs.io/docs/en/babel-plugin-transform-react-jsx#fragments).
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/jsxFragmentFactory.md]:
```suggestion
Esta opção pode ser usada por arquivo, muito semelhante a [diretiva `/** @jsxFrag h */` do Babel](https://babeljs.io/docs/en/babel-plugin-transform-react-jsx#fragments).
```

File: packages/tsconfig-reference/copy/pt/options/strictNullChecks.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
+---
+display: "Strict Null Checks"
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/strictNullChecks.md]:
```suggestion
display: "Checagem estrita de nulos"
```

File: packages/tsconfig-reference/copy/pt/options/strictNullChecks.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+oneline: "Certifique-se de que a nulidade é respeitada no verificador de tipo"
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/strictNullChecks.md]:
```suggestion
oneline: "Certifique-se de que a nulidade é respeitada ao se verificar os tipos"
```

File: packages/tsconfig-reference/copy/pt/options/strictNullChecks.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+---
+
+Quando `strictNullChecks` é `false`, `null` e `undefined` são efetivamentes ignorados pela linguagem. Isso pode levar a erros inesperados no tempo de execução.
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/strictNullChecks.md]:
```suggestion
Quando `strictNullChecks` é `false`, `null` e `undefined` são efetivamentes ignorados pela linguagem. Isso pode levar a erros inesperados em tempo de execução.
```

File: packages/tsconfig-reference/copy/pt/options/strictNullChecks.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+
+Quando `strictNullChecks` é `true`, `null` e `undefined` têm seus próprios tipos distintos e você obterá um erro de tipo se tentar usá-los onde um valor concreto é esperado.
+
+Por exemplo, com este código TypeScript, `users.find` você não tem garantia de que realmente encontrará um usuário, mas você pode escrever o código como se ele fosse:
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/strictNullChecks.md]:
```suggestion
Por exemplo, com este código TypeScript, `usuarios.find` não tem garantia de que realmente encontrará um usuário, mas você pode escrever o código como se ele fosse:
```

File: packages/tsconfig-reference/copy/pt/options/strictNullChecks.md [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+
+```ts twoslash
+// @strictNullChecks: false
+// @target: ES2015
+declare const loggedInUsername: string;
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/strictNullChecks.md]:
```suggestion
declare const nomeDeUsuarioLogado: string;
```

File: packages/tsconfig-reference/copy/pt/options/strictNullChecks.md [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+
+const users = [
+  { name: "Henrique", age: 12 },
+  { name: "Carol", age: 32 }
+];
+
+const loggedInUser = users.find(u => u.name === loggedInUsername);
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/strictNullChecks.md]:
```suggestion
const usuarioLogado = usuarios.find(u => u.nome === nomeDeUsuarioLogado);
```

File: packages/tsconfig-reference/copy/pt/options/strictNullChecks.md [hunk 10]
```diff
@@ -0,0 +1,56 @@
...
+console.log(loggedInUser.age);
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/strictNullChecks.md]:
```suggestion
console.log(usuarioLogado.idade);
```

File: packages/tsconfig-reference/copy/pt/options/strictNullChecks.md [hunk 11]
```diff
@@ -0,0 +1,56 @@
...
+```
+
+Configurar `strictNullChecks` para `true` irá gerar um erro de que você não garantiu a existência de `loggingInUser` antes de tentar usá-lo.
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/strictNullChecks.md]:
```suggestion
Configurar `strictNullChecks` para `true` irá gerar um erro de que você não garantiu a existência de `usuarioLogado` antes de tentar usá-lo.
```

File: packages/tsconfig-reference/copy/pt/options/strictNullChecks.md [hunk 12]
```diff
@@ -0,0 +1,56 @@
...
+
+```ts twoslash
+// @errors: 2339 2532
+// @target: ES2020
+// @strictNullChecks
+declare const loggedInUsername: string;
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/strictNullChecks.md]:
```suggestion
declare const nomeDeUsuarioLogado: string;
```

File: packages/tsconfig-reference/copy/pt/options/strictNullChecks.md [hunk 13]
```diff
@@ -0,0 +1,56 @@
...
+
+const users = [
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/strictNullChecks.md]:
```suggestion
const usuarios = [
```

File: packages/tsconfig-reference/copy/pt/options/strictNullChecks.md [hunk 14]
```diff
@@ -0,0 +1,56 @@
...
+Quando `strictNullChecks` é `true`, `null` e `undefined` têm seus próprios tipos distintos e você obterá um erro de tipo se tentar usá-los onde um valor concreto é esperado.
+
+Por exemplo, com este código TypeScript, `users.find` você não tem garantia de que realmente encontrará um usuário, mas você pode escrever o código como se ele fosse:
+
+```ts twoslash
+// @strictNullChecks: false
+// @target: ES2015
+declare const loggedInUsername: string;
+
+const users = [
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/strictNullChecks.md]:
```suggestion
const usuarios = [
```

File: packages/tsconfig-reference/copy/pt/options/strictNullChecks.md [hunk 15]
```diff
@@ -0,0 +1,56 @@
...
+// @target: ES2020
+// @strictNullChecks
+declare const loggedInUsername: string;
+
+const users = [
+  { name: "Henrique", age: 12 },
+  { name: "Carol", age: 32 }
+];
+
+const loggedInUser = users.find(u => u.name === loggedInUsername);
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/strictNullChecks.md]:
```suggestion
constusuarioLogado = usuarios.find(u => u.nome === nomeDeUsuarioLogado);
```

File: packages/tsconfig-reference/copy/pt/options/strictNullChecks.md [hunk 16]
```diff
@@ -0,0 +1,56 @@
...
+console.log(loggedInUser.age);
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/options/strictNullChecks.md]:
```suggestion
console.log(usuarioLogado.idade);
```

**Review by khaosdoctor — CHANGES_REQUESTED** [review]

File: packages/tsconfig-reference/copy/pt/options/include.md [hunk 17]
```diff
@@ -0,0 +1,66 @@
+---
+display: "Include"
```
Comment (danilofuchs) [inline packages/tsconfig-reference/copy/pt/options/include.md]:
```suggestion
display: "Incluir"
```
O `display` é um texto que facilita encontrar flags ao dar ctrl+F no site. Devem ser traduzidas também!

File: packages/tsconfig-reference/copy/pt/options/jsxFragmentFactory.md [hunk 18]
```diff
@@ -0,0 +1,54 @@
+---
+display: "jsxFragmentFactory"
```
Comment (danilofuchs) [inline packages/tsconfig-reference/copy/pt/options/jsxFragmentFactory.md]:
```suggestion
display: "Fábrica de fragmentos JSX"
```

**Comment by khaosdoctor** [comment]
ping @hmartiins

**Comment by khaosdoctor** [comment]
I think this PR might have staled. @hmartiins are you still with us?

**Comment by hmartiins** [comment]
yes I am yes, I apologize for the delay, I really hadn't seen your ping, I will review this pull request, I apologize again for the delay

**Review by khaosdoctor — APPROVED** [review]
lgtm

**Comment by khaosdoctor** [comment]
No problems! Just to keep up to date 😄

**Comment by khaosdoctor** [comment]
lgtm

**Pull request merged**